index.html

```diff
@@ -33,11 +33,175 @@ <h2 id="id-my-name">Marc Raphael</h2>
         </p>
 
         <!--Here is the transition to the next section-->
-        <button id="id-start-see-project"> Start collaboration </button>
+        <button type="button" id="id-start-see-project"> Start collaboration </button>
       </section> 
 
       <!--a simple div to make a background image easily and end this section-->
       <div id="id-litle-background"></div>
+
+      <!--this section contain information about projects-->
+      <section id="id-works-section">
```

Comment:
- [ ] The basic rule that you need to keep in mind while using `class`es and `id`s in CSS is that, `id` is used for single elements that appear on the page only once (e.g. header, footer, menu), whereas `class` is used for single or multiple elements that appear on the page for once or more than once. The entire point for the styles is for them to be re-usable. So limit the amount of `id`s you use and use them only when necessary. It may seem like there are no difference, but your code will not pass validation if you use the sam `id` on more than 1 element. 
You can refer to [this article](https://css-tricks.com/the-difference-between-id-and-class/) to know more about them.

Reply:
- [ ] This issue is not addressed well. Please see the inline comments under the new review!